content/es/abstraction.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Abstracción
+status: Completed
+category: Propiedad
+---
+
+En el contexto de la informática, una abstracción es una representación que oculta los detalles específicos a un consumidor de [servicios](https://glossary.cncf.io/service/) (un consumidor que es un programa de computadora o un ser humano), haciendo un sistema más genérico y, por lo tanto, más fácil de entender. Un buen ejemplo es el sistema operativo (SO) de su computadora portátil. No necesita saber nada sobre el CPU, la memoria y cómo se manejan los programas, solo necesita operar el sistema operativo y el sistema operativo se ocupa de los detalles. Todos estos detalles se encuentran ocultos detrás de la "cortina" del sistema operativo o la abstracción.
```

Comment:
I suggest to remove hardcoded site domain name:

```suggestion
En el contexto de la informática, una abstracción es una representación que oculta los detalles específicos a un consumidor de [servicios](/service/) (un consumidor que es un programa de computadora o un ser humano), haciendo un sistema más genérico y, por lo tanto, más fácil de entender. Un buen ejemplo es el sistema operativo (SO) de su computadora portátil. No necesita saber nada sobre el CPU, la memoria y cómo se manejan los programas, solo necesita operar el sistema operativo y el sistema operativo se ocupa de los detalles. Todos estos detalles se encuentran ocultos detrás de la "cortina" del sistema operativo o la abstracción.
```

Reply:
Fix on e82e13c76b683cee9be5f5b7fa8816d5c9919522